_config.yml

```diff
@@ -34,7 +34,7 @@ navbar-links:
     - About: "about"
     - Mission: "mission"
     - RSE Groups: "rse-groups"
-    - RSE Map: "map"
+    - RSE Map: "https://usrse.github.io/usrse-map"
```

Comment:
I know this is correct and will work fine.  Can we use {{ site.base_url }} here?  Or if not (never tried this in _config.yml) use us-rse.org instead?

Reply:
No, we cannot.